Build failure in Windows (MinGW-W64) due to missing `<sys/sysctl.h>`

Chipmunk 7.0.0 fails to build in Windows.

The compiler used was MinGW-W64, and the reason appears to be a missing `<sys/sysctl.h>` include in `src/cpHastySpace.c`:

    [ 20%] Building C object src/CMakeFiles/chipmunk.dir/cpHastySpace.c.obj
    C:\Devel\build\Chipmunk-7.0.0\src\cpHastySpace.c:9:24: fatal error: sys/sysctl.h: No such file or directory
     #include <sys/sysctl.h>
                            ^
    compilation terminated.
    src\CMakeFiles\chipmunk.dir\build.make:342: recipe for target 'src/CMakeFiles/chipmunk.dir/cpHastySpace.c.obj' failed
    mingw32-make[2]: *** [src/CMakeFiles/chipmunk.dir/cpHastySpace.c.obj] Error 1
    CMakeFiles\Makefile2:76: recipe for target 'src/CMakeFiles/chipmunk.dir/all' failed
    mingw32-make[1]: *** [src/CMakeFiles/chipmunk.dir/all] Error 2
    makefile:116: recipe for target 'all' failed
    mingw32-make: *** [all] Error 2

Sorry, I wasn't clear about the Mac OS X problem. The #include \<sys/sysctl.h> works fine. The problem on the Mac in the **find_package(Threads)** and **target_link_libraries(chipmunk ${CMAKE_THREAD_LIBS_INIT})**.

That said, #include <unistd.h> doesn't give the results you want:

    [  3%] Building C object src/CMakeFiles/chipmunk_static.dir/cpHastySpace.c.o
    /Users/ezavada/work/pdg/deps/chipmunk/src/cpHastySpace.c:323:3: warning: implicit declaration of function 'sysctlbyname' is invalid in C99
           [-Wimplicit-function-declaration]
                 sysctlbyname("hw.ncpu", &threads, &size, NULL, 0);

This is with 

    Apple LLVM version 6.0 (clang-600.0.56) (based on LLVM 3.5svn)
    Target: x86_64-apple-darwin14.1.0

--- a/src/cpHastySpace.c
+++ b/src/cpHastySpace.c
@@ -6,7 +6,9 @@
 
 #include <pthread.h>
 //#include <sys/param.h >
+#ifndef _WIN32
 #include <sys/sysctl.h>
+#endif
 
 #include "chipmunk/chipmunk_private.h"
 #include "chipmunk/cpHastySpace.h"

